sdk/cognitiveservices/Knowledge.QnAMaker/src/Customizations/Runtime.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker
+{    
+    using Models;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Customized Runtime operations - using V5.0-preivew.1 Knowledgebase operations for generateAnswer and train APIs.
+    /// </summary>
+    public partial class Runtime
```

Comment:
Runtime is a very generic name -- I would recommend you change this to something more specific (if this is a new type).

Reply:
This is introduced for customization and make the SDK fully backward compatible. So, it has to be "Runtime".  Not a new type (present in 2.x.x versions as generated one, from3.0.0- onwards, it is in cstomizations only)

Reply:
Given that this is still in preview I wouldn't bump the major here. If this is backward compatible I would go with 2.0.0-preview.2 or, if you want to bump a number, I would bump minor.

Here are the guidelines we are using for our newer libraries: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-versionnumbers

Reply:
Discussed offline. 
Major version bump is because of the additional features (introduced in public preview API V5.0-preview.1) and small corrections in the previous swagger 4.0. Agreed with the above comment that major version change makes sense.